x-pack/plugins/alerting/server/alerts_client_factory.ts

```diff
@@ -100,7 +85,9 @@ export class AlertsClientFactory {
         excludedWrappers: ['security'],
         includedHiddenTypes: ['alert', 'api_key_pending_invalidation'],
       }),
-      authorization,
+      authorization: this.authorization!.create(request, FEATURE_PRIVILEGE_NAME, [
```

Comment:
Nice, I like that, seems clean. Should we check that `authorization` is indeed there and throw if it's not rather than use the `!`?

Reply:
Updated in this commit: https://github.com/elastic/kibana/pull/99078/commits/ee3f2269ce5c043ab604be40a78d1ec261977ecd